core/modules/system/css/system.admin.css

```diff
@@ -277,6 +277,16 @@ table.screenshot {
   border-top: 1px solid #cdcdcd;
   padding-top: 20px;
 }
+.system-themes-list-disabled-wrapper.flexbox {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-wrap: wrap;
+      -ms-flex-wrap: wrap;
+          flex-wrap: wrap;
+  /*width: calc(100vw - 60px); /* Fix flex-wrap issue with some browsers */
```

Comment:
This is not standard indentation. Also, if the commented-out bit is not required, then it should be removed.